src/IBM.WatsonDeveloperCloud.Assistant.v2/AssistantService.cs

```diff
@@ -26,7 +26,7 @@ namespace IBM.WatsonDeveloperCloud.Assistant.v2
 {
     public partial class AssistantService : WatsonService, IAssistantService
     {
-        const string SERVICE_NAME = "conversation";
+        const string SERVICE_NAME = "assistant";

```

Comment:
The service name is still assistant if what you refer is to the `VCAP_SERVICES` variable. This may not be the case for Unity